orbit/pkg/useraction/mdm_migration_darwin.go

```diff
@@ -374,17 +453,42 @@ func (m *swiftDialogMDMMigrator) renderMigration() error {
 				}
 			}
 
+			switch true {
```

Comment:
https://go.dev/wiki/Switch#missing-expression

```suggestion
			switch {
```